### PR TITLE
 #287 Update release notes to include recommended k8s version

### PR DESCRIPTION
**Pull request by cwoods-cpointe**

### CODE REVIEW

File: DRAFT_RELEASE_NOTES.md [hunk 1]
```diff
@@ -77,4 +84,4 @@ To start your aiSSEMBLE upgrade, update your project's pom.xml to use the 1.9.0
 
 # What's Changed
 - MLFlow Chart version upgraded from `0.2.1` to `1.4.22`. 
-- `aissemble-mlflow` Docker image using MLFlow 2.3.1 was replaced with `bitnami/mlflow:2.15.1-debian-12-r0`  Docker image using MLFlow 2.15.1.
```
Comment (jacksondelametter) [inline DRAFT_RELEASE_NOTES.md:80]: What is the change here?

File: DRAFT_RELEASE_NOTES.md [hunk 2]
```diff
@@ -14,6 +14,13 @@ There are no known issues with the 1.9.0 release.
 | Date<br/>identified | Vulnerability | Severity | Package | Affected <br/>versions | CVE | Fixed <br/>in |
 |---------------------|---------------|----------|---------|------------------------|-----|---------------|
 
+# Recommended Kubernetes Version
+aiSSEMBLE recommends any consumer be on a minimum Kubernetes version of 1.30. This is due to security 
+findings in 1.29. For more information on Kubernetes current security findings, view their 
+[CVE feed](https://kubernetes.io/docs/reference/issues-security/official-cve-feed/). If using EKS, please follow
```
Comment (carter-cundiff) [inline DRAFT_RELEASE_NOTES.md:20]: `If using EKS`. Should this be `If using AWS EKS`?

Comment (cwoods-cpointe) [reply DRAFT_RELEASE_NOTES.md:20]: sure

Comment (carter-cundiff) [reply DRAFT_RELEASE_NOTES.md:20]: Oh I guess EKS is specific to AWS

File: DRAFT_RELEASE_NOTES.md (hunk 1 above)
Comment (cwoods-cpointe) [reply DRAFT_RELEASE_NOTES.md:80]: New line at the end of the file

**Review by carter-cundiff — APPROVED** [review]

File: DRAFT_RELEASE_NOTES.md (hunk 2 above)
Comment (cwoods-cpointe) [reply DRAFT_RELEASE_NOTES.md]: Yeah I think both work. Doesnt hurt to put AWS EKS

**Pull request merged**